deprecated/bookmark+-1.el

```diff
@@ -382,7 +382,7 @@
 ;;    `bmkp-region-jump-other-window', `bmkp-remote-file-jump',
 ;;    `bmkp-remote-file-jump-other-window',
 ;;    `bmkp-remote-non-dir-file-jump',
-;;    `bmkp-temote-non-dir-file-jump-other-window',
+;;    `bmkp-remote-non-dir-file-jump-other-window',
```

Comment:
I'm pretty sure this is a typo because the only sign of this spelling is here, whereas the related things are both listed as deprecated and exist (just like the corrected item).